instrumentation/helpers-mysql/lib/opentelemetry/instrumentation/version.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    VERSION = '0.0.1'
```

Comment:
@ericmustin  - What version should these gems start at? When I skimmed through some of the other libraries on RubyGems, I noticed there isn't a consistent starting value. `0.1.0` feels maybe more appropriate than `0.0.1`. Thoughts?

Reply:
good question, intuition says `0.1.0`. will check

Reply:
yea i think `0.1.0` is the standard for initial releases here, see recent `grape` instrumentation initial release for reference https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/394/files#diff-655c41c1beb01010148b54474ec3d54fb2f06d1c856b3fe9d7b8f59cfd1cff8cR11